toolkit/resources/manifests/package/pkggen_core_aarch64.txt

```diff
@@ -1,5 +1,5 @@
 filesystem-1.1-20.cm2.aarch64.rpm
-kernel-headers-5.15.162.2-1.cm2.noarch.rpm
+kernel-headers-5.15.160.1-1.cm2.noarch.rpm
```

Comment:
I think this maybe a typo .. changing the manifests.

Reply:
Trying to figure out why, I ran two cherry-pick commands on the two PRs... But yes this is unintended.

Reply:
Ok, fixed the commit